docs/Measuring_Resonances.md

```diff
@@ -65,36 +65,28 @@ the system that may damage the electronics.
 
 ## Software installation
 
-Note that resonance measurements and shaper auto-calibration require additional
-software dependencies not installed by default. First, you will have to run on
-your Raspberry Pi the following command:
+Because setting up the software for auto-calibration takes significant
```

Comment:
As a small nit, it is also possible to just measure the resonances of the printer without doing shaper auto-calibration. And the doc name is Measuring_Resonanced.md. So I would update the changes (and perhaps the script name) to reflect that (e.g. here "for resonance measurements").

Reply:
Yes, however the existing instructions have you verify the sensor is working with `MEASURE_AXES_NOISE` before taking measurements which requires `numpy`

Reply:
Yes, of course. But sorry, I did not understand your argument. I agree that everything is needed even for the basic `MEASURE_AXES_NOISE` test.

Maybe I wrote a bit confusingly, I was simply saying that this software is needed not just for auto-calibration (aka `SHAPER_CALIBRATE`), but for any measurement, and I was suggesting to rephrase the sentence to something like:

Because setting up the software for resonance measurements takes significant ...